plugins/webhooks/index.js

```diff
@@ -154,7 +236,15 @@ function pullRequestClosed(options, request, reply) {
         })
         .then(() => request.log(['webhook', hookId, job.id], `${job.name} disabled and archived`)));
 
-    return pipeline.jobs
+    if (!pipeline) {
+        request.log(['webhook', hookId],
+            'Skipping since Pipeline to be closed does not exist');
```

Comment:
'Skipping since Pipeline `for PR` to be closed does not exist' maybe makes more sense?

Do we need to have an equivalent message in `pullRequestOpened` and `pullRequestSync` functions?

Reply:
@tkyi Thank you.
How about 'Skipping  since PR job for `${fullCheckoutUrl}` does not exists.' or 'Skipping since Pipeline ${fullCheckoutUrl} does not exist.'?
This message is showed when 
* There are some other pipelines with branch specific trigger triggered by relevant PR.
* And there is no normal PR job (pipeline) for closed PR. 

When pipeline triggered by a PR (including normal PR trigger and branch specific PR trigger) does not exists, the process is skipped in `pullRequestEvent` function. (skipped [here](https://github.com/screwdriver-cd/screwdriver/pull/1216/files#diff-4bff1c7e1da44fb8e755afbf06260240R375))
So I suppose that we don't need to have an equivalent message in `pullRequestOpened` and `pullRequestSync` functions.

Reply:
Sounds good to me 👍